ui/src/components/executions/Gantt.vue

```diff
@@ -3,7 +3,9 @@
         <table class="table table-sm mb-0">
             <thead>
                 <tr class="bg-light">
-                    <th>{{ duration }}</th>
+                    <th>
+                        <real-time :execution="execution" />
```

Comment:
see comment below about taskrun & execution, it may be better to have 2 props `execution` or `taskrun` to have the detect what is the end

Reply:
handled now

Reply:
I only look at histories. it should be enough no , isn't it ?